### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by jmcpeak** [reduxjs/redux-toolkit#3597] [General]
No response needed, I just wanted to take the time and thank the team and give you guys the kudos you deserve - the documentation of this library is excellent!

The layout, the organization into smaller sections, the shear amount of doc, the attention to small details in the doc...

Thank you, seriously, thank you.

(I know I'm not the only one that thinks this way, everyone that uses this library thinks it)

**Comment by markerikson** [comment]
Thank you, that really means a lot!

tbh _I_ tend to look at our docs and see all the pieces that I know are missing, outdated, or could be improved :)  hugely appreciate the positive feedback!
